guix.scm

```diff
@@ -103,6 +101,8 @@
                        ("r-optparse" ,r-optparse)
                        ("r-qtl" ,r-qtl)
                        ("r-stringi" ,r-stringi)
+                       ("r-wgcna" ,r-wgcna)
+                       ("r-rjson" ,r-rjson)
```

Comment:
**suggestion:** Anytime you modify this file, you should also send a patch to our channel...

Reply:
Thanks for this. My assumption has always been it is updated automatically once you merge to the main  branch merge(more like 
reading the file).